src/mouse_events.cpp

```diff
@@ -1141,4 +1162,19 @@ team & mouse_handler::current_team() {
 }
 
 mouse_handler *mouse_handler::singleton_ = nullptr;
+
+/**  Use this to prevent the mouse to be selecting any unit for
+ * highlighting its movement range. You would want to call this after
+ * highlighting [best] enemy movements in case the mouse cursor landed
+ * on a unit after the menu widget got closed. */
+void mouse_handler::prevent_units_highlight() {
+	preventing_units_highlight_ = true;
+}
+
+/**  When unit highlighting is being prevented, use this when mouse is
+ * finally not over any unit, in order to reenable units highlighting. */
+void mouse_handler::reenable_units_hightlight() {
```

Comment:
I'd prefer just "enable" rather than "reenable" here.

Reply:
Will comply request and will fix typo.